content/en/examples/application/guestbook/redis-leader-deployment.yaml

```
@@ -21,7 +21,7 @@ spec:
     spec:
       containers:
       - name: leader
-        image: "docker.io/redis:6.0.5"
+        image: "registry.k8s.io/redis:v1.7.9"
```

I couldn't find the version for that image
```suggestion
        image: "registry.k8s.io/redis@sha256:cb111d1bd870a6a471385a4a69ad17469d326e9dd91e0e455350cacf36e1b3ee"
```